build/yaml/samples/nsx_v1alpha1_vpcnetworkconfigurations.yaml

```diff
@@ -5,14 +5,14 @@ metadata:
 spec:
   defaultGatewayPath: /infra/tier-0s/t0
   edgeClusterPath: /infra/sites/default/enforcement-points/default/edge-clusters/2d9df59f-6dc6-4911-8865-21fadc23d4da
-  defaultIPv4SubnetSize: 26
-  nsxtProject: proj-1
+  defaultSubnetSize: 26
```

Comment:
please change defaultSubnetSize: to 32, since subnetszie must be power of 2.
so, 26 is invalid.

Reply:
VPC default is 64. Is that WCP uses 32 by default? @lxiaopei @dantingl